server/utils/adjudicationFilterHelper.ts

```diff
@@ -35,26 +35,33 @@ export const uiFilterFromRequest = (req: Request): UiFilter => {
 // get results for all of today, thus if set the 'to' date 2 days ago we should also get the proceeding 2 days.
 // The status default to null, which the api interprets as all statuses.
 export const fillInDefaults = (uiFilter: UiFilter): UiFilter => {
+  /* const allStatuses = [
+    ReportedAdjudicationStatus.SCHEDULED,
+    ReportedAdjudicationStatus.UNSCHEDULED,
+    ReportedAdjudicationStatus.AWAITING_REVIEW,
+    ReportedAdjudicationStatus.RETURNED,
+    ReportedAdjudicationStatus.REJECTED,
+  ] */
```

Comment:
I'm assuming we don't need these anymore seen as they're commented out?

Reply:
remobved